PR for the fixes to the RC subscriptions section (#2915)
RedisLabs/redislabs-docs: content/rc/subscriptions/_index.md

As of February 2021, Redis Enterprise Cloud supports the following subscription plans:

> ### Free plans

Author: I found this was better because it added some menu items to the right. I can remove if needed

> ### Free plans

Reviewer: I don't think the menu items are necessary, and splitting these up into sections makes it seem like the table below is a part of the Annual Plan section. I will say that instead of highlighting each plan type like before the change, it would be better to bold the plan types in the list to make them easier to read.

> ### Free plans

Reviewer: If you really want to have a section for each plan type, you would need to rearrange the section entirely. I have an idea as to how you could do it, but I'm having trouble explaining it in text form, so I'll add a link here to a branch off of this one with the changes that could work once I move stuff around.

> ### Free plans

Author: We can revert it is you want, no worries

As of February 2021, Redis Enterprise Cloud supports the following subscription plans:

- [Free plans](#free-plans)
- [Fixed plans](#fixed-plans)
- [Flexible plans](#flexible-plans)
- [Annual plans](#annual-plans)

> Here's a quick comparison between each plan:

Reviewer 2: ```suggestion
Here's a quick comparison of each plan:
```

> | Feature | [Free plan](#free-plans) | [Fixed plan](#fixed-plans) | [Flexible](#flexible-plans)/<br/>[Annual](#annual-plans) plan |

Reviewer 3: I'm not wild about the links in the table headers. There are already links above, links to the right, and the links themselves link to content that's just right below the table.

To learn more, see [Redis Enterprise Cloud Pricing](https://redislabs.com/redis-enterprise-cloud/pricing/).

### Free plans

Free plans are a tier of Fixed plans designed for training purposes and prototyping. They can be seamlessly upgraded to Fixed plans with no data loss.

### Fixed plans
Fixed plans are cost-efficient and designed for low-throughput scenarios. They support a range of availability, persistence, and backup options.  Pricing supports low throughput workloads.

### Flexible plans

> Flexible plans support more databases (and larger sizes) than lower plans, unlimited connections, and greater throughput. Hosted in dedicated VPCs, they feature high-availability in a single or multi-AZ, active-active geo distribution, Auto-Tiering, clustering, data persistence, and configurable backups.  Pricing is "pay as you go" to support any dataset size or throughput.

Reviewer 2: ```suggestion
Flexible plans support more databases, larger databases, greater throughput, and unlimited connections compared to Fixed plans. Hosted in dedicated VPCs, they feature high-availability in a single or multi-AZ, Active-Active, Auto-Tiering, clustering, data persistence, and configurable backups.  Pricing is "pay as you go" to support any dataset size or throughput.
```

### Annual plans

> Annual plans support the same features as Flexible plans, but at significant savings.  Annual plans also provide Premium support. Further, the underlying commitment applies to all workloads across multiple providers and regions.

Reviewer 2: ```suggestion
Annual plans support the same features as Flexible plans but at significant savings.  Annual plans also provide Premium support. The underlying commitment applies to all workloads across multiple providers and regions.
```
